scripts/build/autotools/CheckHost.m4

```diff
@@ -0,0 +1,54 @@
+AC_CANONICAL_HOST
+
+os_win32=no
+os_netbsd=no
+os_freebsd=no
+os_openbsd=no
+os_hpux=no
+os_linux=no
+os_solaris=no
+os_darwin=no
+os_gnu=no
+
+case "$host" in
+    *-mingw*|*-*-cygwin*)
+        os_win32=yes
+        TARGET_OS=windows
+        ;;
+    *-*-*netbsd*)
+        os_netbsd=yes
+	TARGET_OS=unix
```

Comment:
There are some whitespace issues in this file.

Reply:
It looks like the `TARGET_OS` assignments all used tab indents. Fixing now.